src/main/java/io/github/townyadvanced/flagwar/objects/CellUnderAttack.java

```diff
@@ -275,18 +282,48 @@ public void destroyFlag() {
     }
 
     /**
-     * Function to draw the {@link #hologram}. Retrieves hologram settings via
-     * {@link FlagWarConfig#getHologramSettings()}. Creates a new {@link #hologram} supplying the plugin instance with
-     * {@link FlagWar#getInstance()}, and a temporary {@link org.bukkit.Location} using the {@link #flagLightBlock}.
-     * Disables visibility. Then, iterates through the hologram settings, adding lines corresponding to the line
-     * type and data. Finally, adjusts the location of the hologram according to the height of the {@link #hologram},
-     * and enables its visibility.
+     * Function to draw the {@link #hdHologramsAPI}.
+     * Retrieves hologram settings via {@link FlagWarConfig#getHologramSettings()}.
+     * Will try to draw a hologram with the following priortiy: HolographicDisplays, DescentHolograms
      * */
     public void drawHologram() {
         List<Map.Entry<String, String>> holoSettings = FlagWarConfig.getHologramSettings();
         Location loc = flagLightBlock.getLocation();
-        hologram = HologramsAPI.createHologram(FlagWar.getInstance(), loc);
-        hologram.getVisibilityManager().setVisibleByDefault(false);
+
+        // HolographicDisplays
+        Plugin holographicDisplays = FlagWar.getInstance().getServer().getPluginManager()
+            .getPlugin("HolographicDisplays");
+        if (holographicDisplays != null && holographicDisplays.isEnabled()) {
+            drawHolographicDisplay(loc, holoSettings);
+            return;
+        }
+
+        // DescentHolograms
+        Plugin descentHolograms = FlagWar.getInstance().getServer().getPluginManager().getPlugin("DescentHolograms");
```

Comment:
Testers probably wont have any luck because of this line looking for the wrong name of plugin.